### DISCUSSION TITLE
ad_block.zip

### DISCUSSION THREAD

**Discussion by joako233** [seleniumbase/SeleniumBase#3062] [General]
Hello Michael, when I run my script with the ad_block = True option in Visual Studio Code : 

> with SB(undetectable=True,pls='none',headless=False,block_images=False,ad_block=True) as sb:

, it works perfectly. However, when I create the .exe executable of the script, it generates the following exception. Thanks in advance.
![image](https://github.com/user-attachments/assets/52200ac3-7144-46f9-a4f6-daf455637a04)

**Comment by mdmintz** [comment]
If there's an issue when creating the `.exe`, then that's a [PyInstaller](https://github.com/pyinstaller/pyinstaller) issue. (Not a SeleniumBase one)
